src/pages/devtools/themes/natsuiro/natsuiro.html

```diff
@@ -32,16 +32,16 @@
 			<!-- Module: Activity Box -->
 			<div class="module activity">
 				<div class="activity_tabs">
-					<div id="atab_basic" class="activity_tab hover active i18n" data-target="basic">Basic</div>
-					<div id="atab_battle" class="activity_tab hover i18n" data-target="battle">Battle</div>
-					<div id="atab_expeditionPlanner" class="activity_tab hover i18n" data-target="expeditionPlanner">Exped</div>
-					<div id="atab_activity" class="activity_tab i18n" data-target="">Activity</div>
+					<div id="atab_basic" class="activity_tab hover active i18n" data-target="basic">NatsuiroBasic</div>
+					<div id="atab_battle" class="activity_tab hover i18n" data-target="battle">NatsuiroBattle</div>
+					<div id="atab_expeditionPlanner" class="activity_tab hover i18n" data-target="expeditionPlanner">NatsuiroExped</div>
+					<div id="atab_activity" class="activity_tab i18n" data-target="">NatsuiroActivity</div>
 				</div>
 				<div class="activity_body">
 					<div class="activity_box activity_expeditionPlanner">
 						<div class="expPlan_dropdown_menu">
 							<li class="expPlan_dropdown_title">
-								<div class="dropdown_title">Expedition #1</div>
+								<div class="dropdown_title">ExpedList</div>

```

Comment:
I think the ExpedList should go to natsuiro.js
But i'm not really sure though xD
Because at the panel it still says Expedition #1 even after i change terms.json